pkg/enqueue/Symfony/Processor/RunCommandProcessor.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Enqueue\Symfony\Processor;
+
+use Enqueue\Client\CommandSubscriberInterface;
+use Enqueue\Consumption\Result;
+use Interop\Queue\PsrContext;
+use Interop\Queue\PsrMessage;
+use Interop\Queue\PsrProcessor;
+use Symfony\Component\Process\Exception\ProcessFailedException;
+use Symfony\Component\Process\Process;
+
+class RunCommandProcessor implements PsrProcessor, CommandSubscriberInterface
+{
+    /**
+     * @var string
+     */
+    private $projectDir;
+
+    public function __construct($projectDir)
+    {
+        $this->projectDir = $projectDir;
+    }
+
+    public function process(PsrMessage $message, PsrContext $context)
+    {
+        $commandline = $message->getBody();
+
+        $process = new Process('./bin/console '.$commandline, $this->projectDir);
+
+        try {
+            $process->mustRun();
+
+            return Result::ACK;
```

Comment:
should process send a reply back to the caller?  It could be simply a command exit status or event output.